.github/workflows/python-app.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8
+        pip install build
```

Comment:
You don't need the `build` if you are not using `python -m build` in your build. I see below you use `pip` to build the package. So this is not needed.

Reply:
I followed your advise of having seperate build install steps